plugins/poll/assets/javascripts/discourse/components/poll-buttons-dropdown.gjs

```diff
@@ -35,6 +35,20 @@ const buttonOptionsMap = {
     icon: "lock",
     action: "toggleStatus",
   },
+  showAbsolute: {
+    className: "btn-default show-absolute",
+    label: "poll.show-absolute.label",
+    title: "poll.show-absolute.title",
+    icon: "info",
+    action: "toggleDisplayMode",
+  },
+  showPercentage: {
+    className: "btn-default show-percentage",
```

Comment:
where are these css classes being used? show-percentage show-absolute?

Reply:
Wow thanks for your reminder. I just realized that I should add two lines of css here